layernorm example

bwasti opened the pull request:
Summary: Example usage of the new caffe2 integration

Differential Revision: D13408546

houseroad reviewed (commented):
Great process!
Btw, can we have a tutorial (maybe internal), which shows how to bind a caffe2 op to torch._ops.caffe2?

bwasti committed:
Add correct Input() shim to caffe2 operator impl

Summary:
This enables full shimming of the operator (previously it was only
Output() shimmed).

Differential Revision: D13468241

fbshipit-source-id: 4fe988c2af7d033d30dba212db88da304fc7eeb8

bwasti committed:
plug caffe2 into jit

Differential Revision: D13740752

fbshipit-source-id: 01b8f63d0b9d94cab9629fae9fc3e18eaf240e71

bwasti committed:
Add RunOperator for using FunctionSchema registered ops easily in caffe2

Differential Revision: D13468240

fbshipit-source-id: 1141468bac313699d352cc23d7569f12980e43de

bwasti committed:
layernorm example (#16350)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/16350

Example usage of the new caffe2 integration

Reviewed By: smessmer

Differential Revision: D13408546

fbshipit-source-id: 63132fc703989fdf2d39d53167f7aae386a59047
